tests/integration/security/rbac/v2/basic/testdata/istio-rbac-grpc-rules.yaml.tmpl

```diff
@@ -0,0 +1,71 @@
+# istio-rbac-grpc-rules.yaml to enforce access control for gRPC services using Istio RBAC v2 rules.
+
+# For service a:
+# * Allow b to call a's Echo method.
+# * Do not allow c to talk to a (since NonExistingMethod does not exist).
```

Comment:
The second condition can be removed. Currently we only have "allow" rules. Everything except the two "allows" are denied.

Reply:
This is not really allow or deny (as allow or deny in AuthorizationPolicy), but this defines a grpc method that does not exist in the server, thought I don't think it makes a lot of sense to test it. I will remove it.

Reply:
Sorry, the test case is valid. Maybe the comment is a bit misleading. You can say, 

In AuthorizationPolicy, test the following scenario:
* Refer to a ServiceRole that contains a non-exist method.

Reply:
I don't think this test case is applicable to what we're testing so I removed it (since we never call `NonExistingMethod` in the test anyway).